### PR TITLE
Some cleanups

### PR DESCRIPTION
**Pull request by threepointone**
- update package versions where possible
- run lint across the codebase and do fixes

### CODE REVIEW

**Comment by threepointone** [comment]
lol I added a github action but... 
image

File: public/mockServiceWorker.js [hunk 1]
```diff
@@ -2,286 +2,283 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (2.0.11).
```
Comment (threepointone) [inline public/mockServiceWorker.js:5]: this file was changed automatically, nothing by me

File: app/hooks/useCopyToClipboard.tsx [hunk 2]
```diff
@@ -4,14 +4,20 @@ export default function useCopyToClipboard(delay = 2000) {
...
-	}, [])
+	const copy = useCallback(
+		(value: string) => {
+			navigator.clipboard.writeText(value)
+			setCopied(true)
+			clearTimeout(timeoutRef.current)
+			timeoutRef.current = window.setTimeout(() => {
+				setCopied(false)
+			}, delay)
+			return () => {
```
Comment (threepointone) [inline app/hooks/useCopyToClipboard.tsx]: wait this is silly, fixing

Comment (threepointone) [reply app/hooks/useCopyToClipboard.tsx]: fixed

**Review by third774 — APPROVED** [review]

File: app/components/TextArea.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { cn } from '~/utils/style'
 export const TextArea = forwardRef<
 	HTMLTextAreaElement,
 	JSX.IntrinsicElements['textarea']
->(({ className, ...rest }, ref) => (
+>(({ className, ...rest }, _ref) => (
```
[the remark this answers is not in the document]
Comment (threepointone) [reply app/components/TextArea.tsx:7]: lol ok I'll do that

File: app/utils/Peer.client.ts [hunk 4]
```diff
@@ -432,8 +432,8 @@ export default class Peer {
 						let trackPromises = response.tracks.map((track) => {
 							if (track.mid) {
 								return this.#resolveTrack(track.mid)
-							}
-						})
+							} else return undefined
```
Comment (third774) [inline app/utils/Peer.client.ts:435]: Is this a runtime behavior change?

Comment (threepointone) [reply app/utils/Peer.client.ts:435]: no it happens anyway, but this fixes a lint error which asks it to be explicit

**Pull request merged**